tests/test_PR.py

```diff
@@ -112,5 +113,18 @@ def test_PR_529():
     assert video_clip.rotation == 180
 
 
+def test_PR_610():
```

Comment:
Good call

For a test like this, as I did in PR (#589), I think the best move is to name the test `test_composite_video_clip_should_use_max_fps` and put it under `tests/video/compositing/test_CompositeVideoClip`, which mirrors the folder structure of the library.

I think the underlying issues causing `tests/test_PR` to exist are
1. moviepy's tests aren't structured to mirror the folder structure of the library, so it can be difficult for a contributor to tell where to put a new test.
2. Sometimes PRs are very minor changes/bugfixes that one wouldn't normally write a test for (or are hard to write a test for), like fixing the optional progress bar for detect_scenes (#587) . However, because the PR is coming from an outside contributor and not a maintainer addressing a github issue, even if the changes look simple, there needs to be some level of assurance that the change A) does what it says it does and B) doesn't break the current, intended behavior of detect_scenes (especially when there's no existing test for detect_scenes yet, and its a hassle for a maintainer to go and test it manually).